src/discovery.rs

```diff
@@ -320,7 +318,7 @@ where
     ) -> Result<Self, DiscoveryError<RE>>
     where
         HC: Fn(HttpRequest) -> Result<HttpResponse, RE>,
-        RE: Fail,
+        RE: std::error::Error + Send + Sync + 'static,
```

Comment:
I think we can get rid of these `Send` and `Sync` bounds now after https://github.com/ramosbugs/oauth2-rs/pull/109

Reply:
Check! Should i also remove the 3 `Send + Sync`-bounds in `verification.rs` on following lines?https://github.com/ramosbugs/openidconnect-rs/blob/616133adb6f21a66b1aa9ff0dc4641c5beda3ed4/src/verification.rs#L134
https://github.com/ramosbugs/openidconnect-rs/blob/616133adb6f21a66b1aa9ff0dc4641c5beda3ed4/src/verification.rs#L199
https://github.com/ramosbugs/openidconnect-rs/blob/616133adb6f21a66b1aa9ff0dc4641c5beda3ed4/src/verification.rs#L699